gss-api-app/src/test/java/com/devoops/fake/FakeBlackListRepository.java

```diff
@@ -0,0 +1,30 @@
+package com.devoops.fake;
+
+import com.devoops.domain.repository.auth.BlackListRepository;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import org.springframework.context.annotation.Primary;
+import org.springframework.stereotype.Component;
+
+@Primary
+@Component
+public class FakeBlackListRepository implements BlackListRepository {
+
+    private final List<String> blackList = new ArrayList<>();
```

Comment:
```suggestion
    private final Set<String> blackList = new HashSet<>();
```

토큰이 중복된 값이 발생하지 않을 것 같기도 하고, 조회 성능을 위해서 Set 으로 변환하는 것은 어덜까요?

Reply:
라고 쓰고보니까 테스트 코드 였네요ㅋㅋㅋ넘어가시쥬